server/server.go

```diff
@@ -514,6 +509,41 @@ func (s *Server) startAPIServer() error {
 	return nil
 }
 
+// initAuthzEnforcer initializes authorization enforcer based on configuration
+func (s *Server) initAuthzEnforcer() error {
+	if s.config.TLSClientAuthzModel != "" && s.config.TLSClientAuthzPolicy != "" {
+		s.logger.Warn("DeprecationWarning: Using file-based configuration to persist authorization is no longer supported")
+
+		policyEnforcer, err := casbin.NewEnforcer(s.config.TLSClientAuthzModel, s.config.TLSClientAuthzPolicy)
+		if err != nil {
+			return err
+		}
+		err = policyEnforcer.LoadPolicy()
+
+		if err != nil {
+			return errors.Wrap(err, "failed to load authorization permissions")
+		}
+		s.authzEnforcer = &authzEnforcer{enforcer: policyEnforcer}
+
+	} else {
+		// Default ACL-with-superuser model
+		authzModel, err := model.NewModelFromString(DefaultACLAuthzModel)
+		if err != nil {
+			return err
+		}
+
+		// Initialize casbin object without a storage file
+		// as policies are stored directly on Raft
+		authzStorage := fileadapter.NewAdapter("")
```

Comment:
Not specifying a file to store the ACL policies simplifies a lot the design and implementation. It delegates the whole responsibility of consistency and persistence to Raft.

However, it means that if all the cluster goes down or there are multiple failures that goes beyond recovery capacity of Raft, the ACL policies data will be lost.

IMHO, this is something acceptable for this version, as such incident is considered to be a serious one (e.g: the whole cluster is down), and should be treated outside of the internal implementation. For example, from SysAdmin view point, a regular snapshot via `ListPolicy` is a workaround.